app/commands/tooling_job/cancel.rb

```diff
@@ -0,0 +1,25 @@
+module ToolingJob
+  class Cancel
+    include Mandate
+
+    initialize_with :iteration_uuid
+
+    def call
+      iteration = Iteration.find_by!(uuid: iteration_uuid)
+
+      RestClient.post "#{orchestrator_url}/iterations/cancel", {
+        iteration_uuid: iteration_uuid
+      }
```

Comment:
Rather than this, it should update the job's `job_status` in dynamodb to be `cancelled`. (This probably changed since we originally discussed this 🙂)